packages/expo-dev-launcher/android/src/main/java/expo/modules/devlauncher/launcher/DevLauncherActivity.kt

```diff
@@ -51,10 +53,10 @@ class DevLauncherActivity : ReactActivity(), ReactInstanceManager.ReactInstanceE
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
 
-    contentView = findViewById(android.R.id.content) ?: return
-    splashScreen = DevLauncherSplashScreenProvider()
-      .attachSplashScreenViewAsync(this)
-    searchForRootView()
+//    contentView = findViewById(android.R.id.content) ?: return
+//    splashScreen = DevLauncherSplashScreenProvider()
+//      .attachSplashScreenViewAsync(this)
+//    searchForRootView()
```

Comment:
It is not needed anymore?

Reply:
I was having some initial layout issues w/ dev launcher, I think this has something to do with it. also I don't think splash screen is used on iOS at all so I figure it would be a good idea to strip this out and rethink splash at some point